examples/lujain_work_area/genz-3.py

```diff
@@ -0,0 +1,5 @@
+def reverse_list(n):
+  return n[::-1]
+
+list_of_items=["apple","orange","pineapple","kiwi"]
```

Comment:
According to PEP-8, We give a space after comma , and also space before and after an = sign